tcms/testplans/views.py

```diff
@@ -152,6 +154,8 @@ def render_to_response(self, context, **response_kwargs):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        context['statues'] = TestCaseStatus.objects.all()
+        context['priorities'] = Priority.objects.all()
```

Comment:
Filter is_active = True. Inactive Priorities should be display-only on objects which already use them but not selectable. Also used like that elsewhere in the code base.